index.html

```diff
@@ -360,6 +360,37 @@ <h2>Skills</h2>
             </ul>
         </div>
     </section>
+    <!-- End Section part -->
+
+    <!-- Start Contact form -->
+
+    <div class="contact-form">
+        <div class="formbg" alt="form background"></div>
+        <div class="form-header">
+          <div class="form-header__title">
+                <h4>Contact me</h4>
+            </div>
+            <div class="form-header__text">
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                I’d love to help with it!</p>
+         </div>
+        </div>
+        <div class="inputs">
+            <form action="" class="form">
```

Comment:
Please add action and method attributes and set up your formspree account and get a unique id to get messages from your form. 
The action value should be similar to https://formspree.io/f/{form_id}
method value equals POST

Please check the guideline on[ this page ](https://formspree.io/html/)to add your formspree account to the form.